src/types/TodoError.ts

```diff
@@ -0,0 +1,6 @@
+export enum TodoError {
+  LOAD = 'Unable to load a todo',
```

Comment:
It's better to use more descriptive error messages, such as 'Failed to load the todo item.'